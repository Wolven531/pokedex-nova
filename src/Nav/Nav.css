nav {
	display: flex;
	flex-direction: column;
	grid-column: 2;
	grid-row: 2;
	justify-content: center;
}

nav ul {
	align-content: stretch;
	align-items: center;
	align-self: stretch;
	display: flex;
	justify-content: space-evenly;
	list-style-type: none;
	margin-block-start: 0;
	margin-block-end: 0;
	margin-inline-start: 0;
	margin-inline-end: 0;
	padding-inline-start: 0;
}
nav ul li {
	display: inline-block;
}
