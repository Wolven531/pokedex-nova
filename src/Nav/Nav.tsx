import React, { FC } from 'react'
import { Link } from 'react-router-dom'
import './Nav.css'

export const Nav: FC<any> = () => {
	return (
		<nav>
			<ul>
				<li>
					<Link to="/">Home</Link>
				</li>
				<li>
					<Link to="/other">Other</Link>
				</li>
			</ul>
		</nav>
	)
}
