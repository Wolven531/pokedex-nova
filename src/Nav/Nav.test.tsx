import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import { render, RenderResult } from '@testing-library/react'
import { Nav } from './Nav'

describe('Nav', () => {
	let comp: RenderResult

	beforeEach(() => {
		comp = render(
			<MemoryRouter initialEntries={['/']} initialIndex={0}>
				<Nav />
			</MemoryRouter>,
		)
	})

	it('matches snapshot', () => {
		expect(comp.asFragment()).toMatchSnapshot()
	})
})
