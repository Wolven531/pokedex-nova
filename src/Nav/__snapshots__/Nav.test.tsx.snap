// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav matches snapshot 1`] = `
<DocumentFragment>
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/other"
        >
          Other
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
